cyctl/internal/create/template_auth_rules.go

```diff
@@ -24,16 +26,89 @@ cyctl create templateauthrule demo-templateauthrule --repo='https://github.com/c
 )
 
 var (
-	username string
-	password string
+	username     string
+	password     string
+	usernameName string
+	passwordName string
+	usernameKey  string
+	passwordKey  string
```

Comment:
Can you remove these four from here. `username` and `password` are here because those are used to bind flag values

Reply:
In this I have a doubt here that we are taking the individuals and should we need to add the values the same as username secret name: username secret key , and password secret name: password secret key

Reply:
@petar-cvit could you check it

Reply:
@petar-cvit ??????